File: Linguagem C Exercicios/C-Sequencial/CONSUMO.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    double km,combustivel,consumoMedio = 0 ;

    printf("Distancia percorrida: ");
    scanf("%lf",&km);
    printf("Combustivel gasto: ");
    scanf("%lf",&combustivel);

    consumoMedio = km / combustivel;

    printf("Consumo medio: %.3lf",consumoMedio);

    return 0;
}

File: Linguagem Java/Exercicios/Consumo.java
import java.util.Locale;
import java.util.Scanner;

public class Consumo {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double distancia,combustivel,consumo;
		
		System.out.print("Distancia percorrida: ");
		distancia = sc.nextDouble();
		System.out.print("Combustivel gasto: ");
		combustivel = sc.nextDouble();
		
		consumo = distancia / combustivel;
		
		System.out.println("Consumo medio = "+String.format("%.3f", consumo));
		
		sc.close();

	}

}
